store/pruning/manager_test.go

```diff
@@ -247,6 +300,12 @@ func TestHandleSnapshotHeight_LoadFromDisk(t *testing.T) {
 		loadedSnapshotHeights, err = pruning.LoadPruningSnapshotHeights(db)
 		require.NoError(t, err)
 		require.Equal(t, expected, len(loadedSnapshotHeights), snapshotHeightStr)
+
+		// Change snapshot interval after some height
+		if snapshotHeight == 40 {
+			snapshotInterval = 20
+			manager.SetSnapshotInterval(20)
+		}
```

Comment:
same here. these kinda seem like magic numbers - a bit hard to follow

Reply:
Hi, I have updated both tests to have parameters for these numbers.